source/unified-test-format/unified-test-format.rst

```diff
@@ -3,13 +3,13 @@ Unified Test Format
 ===================
 
 :Spec Title: Unified Test Format
-:Spec Version: 1.2.3
+:Spec Version: 1.3.0
```

Comment:
Please remember to add a changelog entry for this change.

Reply:
Done